islandora_videojs.module

```diff
@@ -75,6 +75,15 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (variable_get('islandora_videojs_responsive', TRUE)) {
     drupal_add_css(drupal_get_path('module', 'islandora_videojs') . '/css/videojs_responsive.css', array('group' => CSS_DEFAULT, 'every_page' => FALSE));
   }
+  $object = islandora_object_load($variables['params']['pid']);
+  $variables['transcript'] = FALSE;
```

Comment:
Functionality wise this seems to my eyes however the [`ds-composite-model`](https://github.com/Islandora/islandora_solution_pack_video/blob/7.x/xml/islandora_video_ds_composite_model.xml) for the video SP doesn't define a `TRANSCRIPT` datastream and no references to transcript within the video SP. Not entirely sure what kind of precedent exists for something like this improvement at this time within Islandora 7.

Reply:
I would say a followup change to the video and audio SPs would be warranted, but not necessary -- this adds a capability to the viewer without actually affecting anyone who doesn't want to use it.

Reply:
Given how little development attention is paid to 7.x, I would not advise blocking a harmless improvement like this for such reasons.